packages/ckeditor5-list/tests/documentlist/documentlistseparator.js

```diff
@@ -0,0 +1,300 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* globals document */
+import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
+import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
+import { getData as getModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view';
+import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
+
+import { DocumentList, DocumentListProperties, DocumentListSeparator } from '../../src';
+
+import stubUid from './_utils/uid';
+
+describe( 'DocumentListSeparator', () => {
+	let editorElement, editor, model, view;
+
+	testUtils.createSinonSandbox();
+
+	beforeEach( async () => {
+		editorElement = document.createElement( 'div' );
+		document.body.appendChild( editorElement );
+
+		editor = await ClassicTestEditor.create( editorElement, {
+			plugins: [
+				Paragraph,
+				DocumentList,
+				DocumentListSeparator
+			]
+		} );
+
+		model = editor.model;
+		view = editor.editing.view;
+
+		stubUid();
+	} );
+
+	afterEach( async () => {
+		if ( editorElement ) {
+			editorElement.remove();
+		}
+
+		if ( editor ) {
+			await editor.destroy();
+		}
+	} );
+
+	it( 'should have pluginName', () => {
+		expect( DocumentListSeparator.pluginName ).to.equal( 'DocumentListSeparator' );
+	} );
+
+	it( 'should be loaded', () => {
+		expect( editor.plugins.get( DocumentListSeparator ) ).to.be.instanceOf( DocumentListSeparator );
+	} );
+
+	it( 'shoud set proper schema rules', () => {
+		expect( model.schema.isRegistered( 'listSeparator' ) ).to.equal( true );
+	} );
+
+	describe( 'upcast', () => {
+		it( 'inserts "listSeparator" element between two "ul" lists in model', () => {
+			editor.setData(
+				'<ul>' +
+				'<li>One</li>' +
+				'</ul>' +
+				'<ul>' +
+				'<li>Two</li>' +
+				'</ul>'
```

Comment:
Please fix the code formatting here and in other tests.

Reply:
Done :heavy_check_mark: